src/index.html

```diff
@@ -1,13 +1,541 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Bakerlab</title>
+    <link rel="icon" href="images/favicon.png">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
+    <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Hello Mate Academy</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+
+  <body class="page__body">
+    <nav class="page__nav nav">
+      <a href="#" class="logo"></a>
+
+      <a
+        href="#menu"
+        class="nav__icon icon icon--menu">
+      </a>
+
+      <nav class="nav__bar">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#products" class="nav__link">Products</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#about-us" class="nav__link">About Us</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#blog" class="nav__link">Blog</a>
+          </li>
+
+          <li class="nav__item-none">
+            <a href="tel:(629) 555-0129" class="button-menu button-menu--icon">
+              Contact us
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </nav>
+
+    <header class="page__section header">
+      <div class="container">
+        <div class="header__content">
+          <div class="grid">
+            <div class="
+              grid__item
+              grid__item--tablet-1-4
+              grid__item--desktop-1-2">
+              <p class="header__subtitle">
+                We bake your fantasies — croissants, cakes, cookies, and cupcakes.
+              </p>
+            </div>
+
+            <div class="
+              grid__item
+              grid__item--tablet-1-8">
+              <h1 class="header__title">
+                Creative bakery
+              </h1>
+            </div>
+
+            <div class="
+            grid__item
+            grid__item--tablet-3-6
+            grid__item--desktop-1-2">
+              <div class="header__button">
+                <a href="#" class="button-header button-header--icon">
+                  Learn more
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="menu__nav nav">
+        <a href="#" class="logo"></a>
+
+        <a
+          href="#"
+          class="icon icon--cross">
+        </a>
```

Comment:
less cumbersome, right?
```suggestion
        <a href="#" class="icon icon--cross" />
```